frontend/src/scenes/Auth/containers/index.tsx

```diff
@@ -9,39 +9,43 @@ import { ILoginUser } from '../../../common/models/auth/ILoginUser';
 import { IRegisterUser } from '../../../common/models/auth/IRegisterUser';
 import { IBindingCallback1 } from '../../../common/models/callback/IBindingCallback1';
 import { addNewUserRoutine, loginUserRoutine } from '../routines';
+import NotFound from '../../NotFound/components/NotFound';
 
 interface IProps {
-    loginUser: IBindingCallback1<ILoginUser>;
-    addNewUser: IBindingCallback1<IRegisterUser>;
+  loginUser: IBindingCallback1<ILoginUser>;
+  addNewUser: IBindingCallback1<IRegisterUser>;
 }
 const Auth: FunctionComponent<IProps> = ({
   loginUser,
   addNewUser
 }: IProps) => (
   <div className={styles.pageLayout}>
     <div className={styles.rightSide}>
-      <Route
-        exact
-        path={Routes.SignIn}
-        render={props => (
-          <SignIn
-            {...props}
-            loginUser={loginUser}
-          />
-        )}
-        key={Routes.SignIn}
-      />
-      <Route
-        exact
-        path={Routes.SignUp}
-        render={props => (
-          <SignUp
-            {...props}
-            addNewUser={addNewUser}
-          />
-        )}
-        key={Routes.SignUp}
-      />
+      <Switch>
+        <Route
+          exact
+          path={Routes.SignIn}
+          render={props => (
+            <SignIn
+              {...props}
+              loginUser={loginUser}
+            />
+          )}
+          key={Routes.SignIn}
+        />
+        <Route
+          exact
+          path={Routes.SignUp}
+          render={props => (
+            <SignUp
+              {...props}
+              addNewUser={addNewUser}
+            />
+          )}
+          key={Routes.SignUp}
+        />
+        <Route path="*" component={NotFound} />
```

Comment:
We need to have the NotFound route only in the root routing component

Reply:
OK.  I'll only leave NotFound in the root route. Without NotFound, when testing, for example, route / authentication / path, it produced an empty page.

Reply:
Then it needs to be fixed but not with the NotFound page. We should have a default route for redirect in such scenes. For example, here the default route should be /sign-in

Reply:
Ok.